cmd/podman/export.go

```diff
@@ -66,5 +70,23 @@ func exportCmd(c *cli.Context) error {
 		return errors.Wrapf(err, "error looking up container %q", args[0])
 	}
 
+	if os.Geteuid() != 0 {
```

Comment:
Should we have a check here to error if the container isn't running before trying to open conmon?

Reply:
right, good point.  I'll also need to check if it works when the container is not running

Reply:
pushed a new version with the fix